src/controllers/Checkout.controller.ts

```diff
@@ -6,20 +6,37 @@ const stripe = new Stripe(process.env.STRIPE_SECRET as string, {
 });
 
 export const checkoutController = {
-  async checkoutPayment(_req: Request, res: Response) {
+  async checkoutPayment(req: Request, res: Response) {
     try {
-      // we still need ID coming from the front-end for this to work
-      const paymentIntent = await stripe.paymentIntents.create({
-        amount: 4000, // generic number
+      const customer = await stripe.customers.create({
+        email: req.body.email,
+        source: req.body.tokenId,
+        description: "Customer from the e-commerce",
+      });
+
+      const amount = req.body.amount;
```

Comment:
@shyakadavis great idea right here, . just have a few suggestions for better clean code check below: 
```suggestion
      const { email, tokenId, amount } = req.body;
      
      const customer = await stripe.customers.create({
        email,
        source: tokenId,
        description: "Customer from the e-commerce",
      });
```

Reply:
Awesome. 🥏